### ISSUE TITLE
Cannot open the viewer when images are put in a sub directory since v0.17.2

### ISSUE THREAD

**Issue by ysgk** [reg-viz/reg-cli#405]
## Describe the bug

The comparison report doesn't work correctly since 0.17.2. The Viewer won't open after clicking a card when the image files are placed in a sub-directory.

## Reproduced step

https://github.com/ysgk/reg-suit-ui-repro

```shell
git clone git@github.com:ysgk/reg-suit-ui-repro.git
cd reg-suit-ui-repro
yarn
yarn reg-suit compare
npx http-server .reg
```

You can click to open the first item but the second won't open.

## Expected behavior

Regardless of the hierarchy the images are placed, the viewer ui can be shown.

## Actual behavior

Nothing happens after clicking a card if the image is placed in a sub-directory.

## The problem
After examining the code, I found that this line was the problem.
https://github.com/reg-viz/reg-cli-report-ui/blob/465b91363112eb4f7d26f70328596809763b993e/src/containers/viewer/ViewerContainer.ts#L117

```js
// rawId: new-sub/sample.png
// id:    new-sub%2Fsample.png

const id = encodeURIComponent(rawId);
const index = allItems.findIndex((entity) => entity.id === id);
```

If the id has a slash, the `findIndex` always returns -1  because `id` is URI-encoded while `entity.id` is not. Actually, `entity.id` is encoded [here](https://github.com/reg-viz/reg-cli/blob/620bef4e57f264e3b8f1ad2494f8884b218f4632/src/report.js#L34-L39) but the slashes are left unencoded.

I didn't understand the intent of the encoding so can you take a look at that?

**Comment by wadackel** [comment]
@ysgk 

Thank you for your report.
I confirmed that the bug can be reproduced with the repository you provided.

I didn't realize that slash wasn't encoded.
(It seems that it started to use encoded characters from [here](https://github.com/reg-viz/reg-suit/issues/124).)

I'm trying to fix it in the next PR.
https://github.com/reg-viz/reg-cli-report-ui/pull/48

**Comment by ysgk** [comment]
I have confirmed that the bug has been fixed in 0.17.3. Thank you!

**Comment by wadackel** [comment]
Thank you for your report ;)
